common/culture/traditions/00_regional_traditions.txt

```diff
@@ -834,21 +860,16 @@ tradition_desert_nomads = {
 	}
 	
 	is_shown = {
-		OR = {
-			like_culture_bedouin_trigger = yes
-			any_parent_culture = {
-				has_cultural_tradition = tradition_desert_nomads
-			}
-		}
+		# Warcraft
+		any_culture_county = {
+			any_county_province = { geographical_region = custom_deserts }
```

Comment:
Double any cycles. Check it in other places please.

Reply:
Kept `any_county_province` for provincial terrain, replaced with `title_province` for `geographical_region`.

Reply:
There's another such case for `tradition_lords_of_the_elephant`.

Reply:
This is how @Gaben1025 implemented innovations. It's already on `1.5-preview-adaptation`.

Reply:
I asked him. Anyway, you should check such cases and replace them with less CPU heavy solutions.